PipeMethodCalls/Endpoints/PipeServer.cs

```diff
@@ -33,6 +33,17 @@ public PipeServer(string pipeName, Func<THandling> handlerFactoryFunc, PipeOptio
 			this.options = options;
 		}
 		
+		/// <summary>
+		/// Initializes a new instance of the <see cref="PipeServer{THandling}"/> class.
+		/// </summary>
+		/// <param name="existingPipe">Instance of a named pipe to upgrade.</param>

```

Comment:
Raw pipe stream to wrap with method call capability.

Reply:
That sounds good.  You've got it 👍